src/app/features/call/CallView.tsx

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-nested-ternary */
+import { Room } from 'matrix-js-sdk';
+import React, { useContext, useMemo, useCallback, useEffect, useRef } from 'react';
+import { Box } from 'folds';
+import { RoomViewHeader } from '../room/RoomViewHeader';
+import { useCallState } from '../../pages/client/call/CallProvider';
+import {
+  PrimaryRefContext,
+  BackupRefContext,
+} from '../../pages/client/call/PersistentCallContainer';
+import { ScreenSize, useScreenSizeContext } from '../../hooks/useScreenSize';
+
+function debounce<F extends (...args: any[]) => any>(func: F, waitFor: number) {
+  let timeoutId: ReturnType<typeof setTimeout> | null = null;
+  return (...args: Parameters<F>): void => {
+    if (timeoutId) {
+      clearTimeout(timeoutId);
+    }
+    timeoutId = setTimeout(() => func(...args), waitFor);
+  };
+}
+
+type OriginalStyles = {
+  position?: string;
+  top?: string;
+  left?: string;
+  width?: string;
+  height?: string;
+  zIndex?: string;
+  display?: string;
+  visibility?: string;
+  pointerEvents?: string;
+  border?: string;
+};
+
+export function CallView({ room, eventId }: { room: Room; eventId?: string }) {
```

Comment:
Unused param that is normally used for the timeline
```suggestion
export function CallView({ room }: { room: Room }) {
```